vm/src/stdlib/binascii.rs

```diff
@@ -44,9 +44,9 @@ mod decl {
         #[inline]
         pub fn with_ref<R>(&self, f: impl FnOnce(&[u8]) -> R) -> R {
             match self {
-                SerializedData::Bytes(b) => f(b.get_value()),
```

Comment:
You could implement BorrowValue for PyBytesLike, maybe? Maybe with `Borrowed` as an enum `LockOrRef` that looks like:

```rust
enum LockOrRef<'a, T> {
    Ref(&'a T),
    MuLock(PyMutexGuard<'a, T>),
    ReadLock(PyRwLockReadGuard<'a, T>),
}
impl Deref for LockOrRef { ... }
```
Maybe somewhere in rustpython_common?

Reply:
Maybe BorrowValue could even go in rustpython_common? It is very generic.

Reply:
It sounds nice. But I want to concentrate on simple replacement in this PR

Reply:
Sounds good